includes/helpers.inc

```diff
@@ -47,6 +58,63 @@ function _tide_read_config($id, array $locations = []) {
   return $storages[$path]->read($id);
 }
 
+/**
+ * Helper to delete configuration from provided locations.
+ *
+ * CONFIG_SYNC_DIRECTORY is prepended to the beginning of the locations list
+ * to perform lookup in active configuration.
+ *
+ * @param string $id
+ *   Configuration id.
+ * @param array $locations
+ *   Array of paths to lookup configuration files.
+ * @param bool $file_system_delete
+ *   Determine if config file should be deleted from config sync directory.
+ *
+ * @return bool
+ *   Indicates success or failure of the function.
+ *
+ * @throws \Exception
+ *   If configuration file was not deleted successfully.
+ */
+function _tide_delete_config($id, array $locations = [], $file_system_delete = FALSE) {
```

Comment:
If you remove a file from `config/sync` in your commit, it will be removed upon deployment, and the config will be removed during config-import. Deleting config files via code won't work.

Reply:
@sonnykt This did not get actioned. Is this change you've requested something you can do so we can get this merged?